mlrun/projects/project.py

```diff
@@ -3795,7 +3795,7 @@ def list_model_endpoints(
         top_level: bool = False,
         uids: Optional[list[str]] = None,
         latest_only: bool = False,
-        tsdb_metrics: bool = True,
+        tsdb_metrics: Optional[str] = None,
```

Comment:
missing from the docstring (it was already missing before your PR, but you can copy and paste from what you’ve already written)